<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="m_pMapEditor" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMapScreenshot"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="openDatabase"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDatabase"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionMapScreenshot">
   <property name="text">
    <string>map screenshot</string>
   </property>
   <property name="toolTip">
    <string>Stores screenshot of currentmap in clipboard.</string>
   </property>
  </action>
  <action name="openDatabase">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
